### ISSUE TITLE
508-defect-3 [COGNITION]: Wizard, links should be styled as links

### ISSUE THREAD

**Issue by joshkimux** [department-of-veterans-affairs/va.gov-team#22733]
# [508-defect-3](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/platform/accessibility/guidance/defect-severity-rubric.md#508-defect-3)

## Feedback framework

- **❗️ Must** for if the feedback must be applied 
- **⚠️ Should** if the feedback is best practice 
- **✔️ Consider** for suggestions/enhancements 

## Definition of done

1. Review and acknowledge feedback.
1. Fix and/or document decisions made.
1. Accessibility specialist will close ticket after reviewing documented decisions / validating fix.

<hr/>

## Point of Contact

**VFS Point of Contact:** Josh

## User Story or Problem Statement

- As a user, I expect links to be styled as links so that I know I can open them in a new tab.
- As a voice AT user, I expect links to be styled as links so that I can provide an accurate command based on what I see.
- As a screen reader user with low vision, I expect the screen reader to announce what I see on the page.
- As a user, I expect links to be styled consistently as links so that I do not confuse them with buttons.

## Details

The "Submit a Financial Status Report" link within the wizard is styled as a `button` which is materially dishonest. This causes several issues:
- It's materially dishonest and damages habituation on VA.gov 
- Voice assistive tech users may say "Click go to xyz form _button_" which may result in failure as it is tagged as a `a`
- Screen reader users with low vision may become confused at a button being announced as a link, and even more so when navigating by links/buttons using the rotor
- Sighted users may not know that they can open this in a new tab because it looks like a button, especially on mobile which gives a stronger affordance for a tap vs. a tap/hold

This is a sitewide long-tail effort. [To better track button/link inconsistencies, please view this Mural.](https://app.mural.co/invitation/mural/vsa8243/1610052905994?sender=u2134c22982ad9c7b41798011&key=87284ca2-fd99-431d-b086-5890abf480ac).

## Acceptance Criteria

- [ ] Links are styled as links

## Environment

* Operating System: all
* Browser: any
* Screenreading device: any
* Server destination: staging

## Solution 

Use the new action link component in the VA design system

## WCAG or Vendor Guidance (optional)

* [Adam Silver: But sometimes buttons look like links](https://adamsilver.io/articles/but-sometimes-buttons-look-like-links/)
* [WCAG Success Criterion 3.2.4: Consistent Identification](https://www.w3.org/WAI/WCAG21/Understanding/consistent-identification.html)

## Screenshots or Trace Logs

![image](https://user-images.githubusercontent.com/14154792/113760907-d4b0d280-96e4-11eb-87ff-ee1a07b7911a.png)

**Comment by denisecoveyduc** [comment]
@joshkimux @AngelaFowler82  I am doing a little GH clean up and wondered if you could just take a quick look at this issue for me to validate that this is still a concern? Since (and before I joined the team) the FSR has gone through many changes and this may or may not have already been addressed. Thanks in advance for checking it out for us.

**Comment by AngelaFowler82** [comment]
I don't see a link or button labeled "submit a financial status report." However, the link which appears when you complete the wizard which takes you to the form is labeled "Start your request now," and is in fact coded as a link. If someone could varrify that I'm correctly identifying the item referred to in this ticket, we can close this.

**Comment by joshkimux** [comment]
Hey folks, I gave this a check a second time and it looks like several of these issues still need to be resolved.

On the manage your VA debt page, the "request help with VA debt" button is in fact a link styled as a button. Using the action link component can resolve this issue so that it is link semantically and visually.

<img width="1348" alt="Screen Shot 2022-01-12 at 10 18 21 AM" src="https://user-images.githubusercontent.com/14154792/149215930-f5ad2025-8b8d-4ba9-b215-f99a4badf19f.png">

On the wizard, one of the final choices "start your request now" is also a link styled as a button. Using the action link component can resolve this issue too.

<img width="1427" alt="Screen Shot 2022-01-12 at 10 20 44 AM" src="https://user-images.githubusercontent.com/14154792/149215903-9a2e0351-5bfa-486b-a597-8f708b1a486e.png">

**Comment by denisecoveyduc** [comment]
Thanks to you both. We will look into this one as well. @rileyorr also drawing your attention to this one before we hand it over to FE.

**Comment by denisecoveyduc** [comment]
Shifting this over to our Tech debt epic to ensure we can address this and move it forward. It was originally captured as part of the staging review for the initial FSR launch.

**Comment by HeatherRienks** [comment]
Per @josephrlee this has already been resolved!
